tests/test_logic.py

```diff
@@ -143,9 +143,9 @@ def test_Controller_call_api(homedir, config, mocker, session_maker):
     thread.started.connect.assert_called_once_with(runner.call_api)
     thread.start.assert_called_once_with()
     runner.moveToThread.assert_called_once_with(thread)
-    runner.call_succeeded.connect.call_count == 1
-    runner.call_failed.connect.call_count == 1
-    runner.call_timed_out.connect.call_count == 1
+    assert runner.call_succeeded.connect.call_count == 1
+    assert runner.call_failed.connect.call_count == 1
+    assert runner.call_timed_out.connect.call_count == 0
```

Comment:
I can see we have an unused `call_timed_out` signal - yes, we emit it, but nothing is connected to it, so should probably just get removed (doesn't have to be part of this PR but just making note)

Reply:
Yes! I wasn't 100% sure it wasn't used _anywhere_ and preferred sticking to fixing the assertions, but I totally agree that removing it would be best!